config/webpack.config.js

```diff
@@ -224,7 +224,7 @@ function generateHtmlFiles(buildPath, scaffoldAssets) {
         'polyfills',
         useLocalStylesAndComponents ? null : 'web-components',
         'vendor',
-        useLocalStylesAndComponents ? null : 'style',
```

Comment:
This is just a direct roll back of [this change](https://github.com/department-of-veterans-affairs/vets-website/pull/22667/files#diff-c49a22733230fd83e34ff3bc78014ee352ad46ce10c7c502dd2a7c3982adb0a5R207). Originally, this ternary was intended to allow applications to opt out of the global styles used in src/platform/sitewide/style.scss. However, the one application that was using it no longer is. This doesn't change default behavior.